core/build.gradle.kts

```diff
@@ -68,6 +68,7 @@ dependencies {
     implementation(libs.ktorServerCallLogging)
     implementation(libs.ktorServerContentNegotiation)
     implementation(libs.ktorServerCore)
+    implementation(libs.ktorServerCors)
```

Comment:
Please link to https://ktor.io/docs/server-cors.html in the commit message to explain to readers what CORS is.

Reply:
Could you also briefly explain why this is required, e.g. where do cross origin requests happen?